frontend/src/components/Timeline.vue

```diff
@@ -0,0 +1,60 @@
+<template lang="pug">
+  #app
+    v-app#inspire
+      v-timeline(:dense='$vuetify.breakpoint.smAndDown')
+        v-timeline-item(color='light-blue' fill-dot='' center='')
+          span(slot='opposite')
+            h5.display-1.light-blue--text
```

Comment:
Try to use vuetify Font sizes. https://vuetifyjs.com/en/styles/typography/#font-sizes

Reply:
okay